src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -52,5 +52,16 @@ public class OtlpExporterOptions
         /// Gets or sets the BatchExportProcessor options. Ignored unless ExportProcessorType is Batch.
         /// </summary>
         public BatchExportProcessorOptions<Activity> BatchExportProcessorOptions { get; set; } = new BatchExportProcessorOptions<Activity>();
+
+        /// <summary>
+        /// Gets or sets the metric export interval.
+        /// </summary>
+        public int MetricExportInterval { get; set; } = 1000;
```

Comment:
For now just putting these new metric related options here, though it might make sense to make an `OtlpMetricsExporterOptions` that derives from `OtlpExporterOptions`. Though also there are things like `BatchExportProcessorOptions` on this class that may not make sense in the context of metrics.